drivers/ovsSwitch.go

```diff
@@ -222,11 +222,12 @@ func setLinkMtu(name string, mtu int) error {
 	return netlink.LinkSetMTU(iface, mtu)
 }
 
-// getOvsPostName returns OVS port name depending on if we use Veth pairs
-func getOvsPostName(intfName string) string {
+// getOvsPortName returns OVS port name depending on if we use Veth pairs
+// For infra nw, dont use Veth pair
+func getOvsPortName(intfName, nwType string) string {
```

Comment:
I think it would be better if we pass a useVethPair argument to this function rather than nwType. Would be nice to avoid propagating the nw type where it is not necessary.